pyins/tests/test_transform.py

```diff
@@ -1,7 +1,12 @@
 import numpy as np
 from numpy.testing import assert_allclose
+import pandas as pd
 from scipy.spatial.transform import Rotation
 from pyins import earth, transform
+from pyins.sim import generate_sine_velocity_motion
```

Comment:
I think it's reasonable to stick to only module imports from pyins (just for some consistency).